### PR TITLE
Added accessibility documentation for components

### LINKED ISSUE #1540: Add accessibility section for `autocomplete`
### Subject

accessibility section for `autocomplete`

#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )
https://www.radix-ui.com/primitives/docs/components/accordion#accessibility
https://mantine.dev/core/autocomplete/#accessibility

#### Include the following content
* keboard interaction
* aria, tabindex property are supported

#### reference
https://github.com/yamada-ui/yamada-ui/issues/2050

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #1540

## Description

Added accessibility documentation for components.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: docs/contents/components/forms/autocomplete/accessibility.ja.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+| ---------------------------------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `AutocompleteField` <Tag>内部</Tag>            | `role="combobox"`          | コンボボックスであることを示します。                                                                                |
+|                                                | `aria-haspopup="listbox"`  | リストボックスが存在することを示します。                                                                            |
+|                                                | `aria-autocomplete="list"` | 入力値に関連する候補リストを提案することを示します。                                                                |
+|                                                | `aria-activedescendant`    | フォーカスされた項目の`id`を設定します。                                                                            |
+|                                                | `aria-expanded`            | リストボックスが開いている場合は"true"を設定し、閉じている場合は"false"を設定します。                               |
```
Comment (chida09) [inline docs/contents/components/forms/autocomplete/accessibility.ja.mdx:55]:
```suggestion
|                                                | `aria-expanded`            | リストボックスが開いている場合は`"true"`を設定し、閉じている場合は`"false"`を設定します。                               |
```

File: docs/contents/components/forms/multi-autocomplete/accessibility.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+description: >-
+  `MultiAutocomplete` is a component used to display suggestions based on user
+  text input and to obtain multiple values.
+package_name: "@yamada-ui/autocomplete"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+The `MultiAutocomplete` follows the[WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```
Comment (chida09) [inline docs/contents/components/forms/multi-autocomplete/accessibility.mdx:12]:
```suggestion
The `MultiAutocomplete` follows the [WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```

File: docs/contents/components/forms/autocomplete/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+description: >-
+  `Autocomplete` is a component used to display suggestions in response to user
+  text input.
+package_name: "@yamada-ui/autocomplete"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+The `Autocomplete` follows the[WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```
Comment (chida09) [inline docs/contents/components/forms/autocomplete/accessibility.mdx:12]:
```suggestion
The `Autocomplete` follows the [WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```

File: docs/contents/components/forms/autocomplete/accessibility.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+| -------------------------------------------------- | -------------------------- | ------------------------------------------------------------------------------------- |
+| `AutocompleteField` <Tag>Internal</Tag>            | `role="combobox"`          | Indicates that this is a combobox.                                                    |
+|                                                    | `aria-haspopup="listbox"`  | Indicates that a listbox exists.                                                      |
+|                                                    | `aria-autocomplete="list"` | Indicates that the component proposes a list of values related to the input value.    |
+|                                                    | `aria-activedescendant`    | Set to the `id` of the focused item.                                                  |
+|                                                    | `aria-expanded`            | Set to "true" when the listbox is open, "false" when closed.                          |
```
Comment (chida09) [inline docs/contents/components/forms/autocomplete/accessibility.mdx:57]:
```suggestion
|                                                    | `aria-expanded`            | Set to `"true"` when the listbox is open, `"false"` when closed.                          |
```

File: docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+| ---------------------------------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| `MultiAutocompleteField` <Tag>内部</Tag>       | `role="combobox"`          | コンボボックスであることを示します。                                                                                     |
+|                                                | `aria-haspopup="listbox"`  | リストボックスが存在することを示します。                                                                                 |
+|                                                | `aria-autocomplete="list"` | 入力値に関連する候補リストを提案することを示します。                                                                     |
+|                                                | `aria-activedescendant`    | フォーカスされた項目の`id`を設定します。                                                                                 |
+|                                                | `aria-expanded`            | リストボックスが開いている場合は"true"を設定し、閉じている場合は"false"を設定します。                                    |
```
Comment (chida09) [inline docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx:55]:
```suggestion
|                                                | `aria-expanded`            | リストボックスが開いている場合は`"true"`を設定し、閉じている場合は`"false"`を設定します。                                    |
```

File: docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+| `AutocompleteClearIcon` <Tag>内部</Tag>        | `role="button"`            | ボタンであることを示します。                                                                                             |
+|                                                | `aria-label`               | `"Clear value"`を設定します。                                                                                            |
+| `MultiAutocompleteList` <Tag>内部</Tag>        | `role="listbox"`           | リストボックスであることを示します。                                                                                     |
+|                                                | `id`                       | `MultiAutocompleteField`と関連付けるために使用する`id`。                                                                 |
+|                                                | `aria-multiselectable`     | 複数選択可能であることを示します。                                                                                       |
+|                                                | `aria-hidden`              | リストボックスが開いている場合は"true"を設定し、閉じている場合は"false"を設定します。                                    |
```
Comment (chida09) [inline docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx:66]:
```suggestion
|                                                | `aria-hidden`              | リストボックスが開いている場合は`"true"`を設定し、閉じている場合は`"false"`を設定します。                                    |
```

File: docs/contents/components/forms/multi-autocomplete/accessibility.mdx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+| -------------------------------------------------- | -------------------------- | ------------------------------------------------------------------------------------------ |
+| `MultiAutocompleteField` <Tag>Internal</Tag>       | `role="combobox"`          | Indicates that this is a combobox.                                                         |
+|                                                    | `aria-haspopup="listbox"`  | Indicates that a listbox exists.                                                           |
+|                                                    | `aria-autocomplete="list"` | Indicates that the component proposes a list of values related to the input value.         |
+|                                                    | `aria-activedescendant`    | Set to the `id` of the focused item.                                                       |
+|                                                    | `aria-expanded`            | Set to "true" when the listbox is open, "false" when closed.                               |
```
Comment (chida09) [inline docs/contents/components/forms/multi-autocomplete/accessibility.mdx:57]:
```suggestion
|                                                    | `aria-expanded`            | Set to `"true"` when the listbox is open, `"false"` when closed.                               |
```

File: docs/contents/components/forms/multi-autocomplete/accessibility.mdx [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+| `AutocompleteClearIcon` <Tag>Internal</Tag>        | `role="button"`            | Indicates that this is a button.                                                           |
+|                                                    | `aria-label`               | `"Clear value"` is set.                                                                    |
+| `MultiAutocompleteList` <Tag>Internal</Tag>        | `role="listbox"`           | Indicates that this is a listbox.                                                          |
+|                                                    | `id`                       | The `id` used to associate with `MultiAutocompleteField`.                                  |
+|                                                    | `aria-multiselectable`     | Indicates that multiple selection is possible.                                             |
+|                                                    | `aria-hidden`              | Set to "true" when the listbox is open, "false" when closed.                               |
```
Comment (chida09) [inline docs/contents/components/forms/multi-autocomplete/accessibility.mdx:68]:
```suggestion
|                                                    | `aria-hidden`              | Set to `"true"` when the listbox is open, `"false"` when closed.                               |
```

File: docs/contents/components/forms/autocomplete/accessibility.mdx (hunk 4 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/autocomplete/accessibility.mdx:57]: Fixed.

File: docs/contents/components/forms/autocomplete/accessibility.mdx (hunk 3 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/autocomplete/accessibility.mdx:12]: Fixed.

File: docs/contents/components/forms/multi-autocomplete/accessibility.mdx (hunk 2 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-autocomplete/accessibility.mdx:12]: Fixed.

File: docs/contents/components/forms/multi-autocomplete/accessibility.mdx (hunk 8 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-autocomplete/accessibility.mdx:68]: Fixed.

File: docs/contents/components/forms/multi-autocomplete/accessibility.mdx (hunk 7 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-autocomplete/accessibility.mdx:57]: Fixed.

File: docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx (hunk 5 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx:55]: Fixed.

File: docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx (hunk 6 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-autocomplete/accessibility.ja.mdx:66]: Fixed.

**Review by chida09 — APPROVED** [review]

**Pull request merged**